<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuADM</class>
 <widget class="QWidget" name="MenuADM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>731</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#logout_button {
	background-color: rgba(0, 0, 0, 0);
	color:rgba(85, 98, 112, 255);
    border-radius: 20px;
    border: 1px solid rgba(255, 255, 255, 30);
    font-weight: bold;
}

QPushButton#logout_button:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(25, 55, 100, 240),
        stop:1 rgba(220, 220, 220, 80)
    );
	color:rgba(155, 168, 182, 220);
}

QPushButton#logout_button:pressed {
	padding-left:5px;
	padding-top:5px;
	color:rgba(115, 128, 142, 255);
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(10, 25, 50, 230),
        stop:1 rgba(180, 180, 180, 50)
    );
}

QPushButton#cliente_button, QPushButton#relatorio_button, QPushButton#produtos_button, QPushButton#funcionario_button {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(15, 35, 70, 120), 
        stop:1 rgba(200, 200, 200, 40)
    );
    color: rgba(255, 255, 255, 180);  
    border-radius: 5px;
    border: 1px solid rgba(255, 255, 255, 30);
    font-weight: bold;
}

QPushButton#cliente_button:hover, QPushButton#relatorio_button:hover, QPushButton#produtos_button:hover,
QPushButton#funcionario_button:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(25, 55, 100, 150),
        stop:1 rgba(220, 220, 220, 50)
    );
}

QPushButton#cliente_button:pressed, QPushButton#relatorio_button:pressed, QPushButton#produtos_button:pressed,
QPushButton#funcionario_button:pressed {
    padding-left: 3px;
    padding-top: 3px;
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:1, y2:1,
        stop:0 rgba(10, 25, 50, 150),
        stop:1 rgba(180, 180, 180, 30)
    );
}
</string>
   </property>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>711</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/log_background.png);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fundo_transparente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>691</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba( 0 ,0, 0, 100 );
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>100</y>
      <width>450</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>691</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba( 0 ,0, 0, 100 );
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Logout</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/ic_logout.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cliente_button">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>45</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="funcionario_button">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>45</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Funcion√°rio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>390</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 180);  </string>
    </property>
    <property name="text">
     <string>Modo Administrador</string>
    </property>
   </widget>
   <widget class="QPushButton" name="produtos_button">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>45</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Produto</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
